content/en/blog/_posts/2021-01-25-Work-with-Persistent-Volume-and-Secrets-for-Deploying-MySQL-on-Kubernetes.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: blog
+title: "Working with PersistentVolume and Secrets for Deploying MySQL on Kubernetes"
+date: 2021-01-25
+slug: work-with-persistent-volume-and-secrets-for-deploying-mysql-on-kubernetes
+---
+
+In most of full-stack applications, there is an authentication system
+for users to register their account and conduct activities in the
+system. The users’ data is usually stored in a SQL or No-SQL databases.
+To setup and deploy MySQL database on Kubernetes, you need to know 3
+major concepts: Persistent Volumes, Secrets and Persistent Volume Claim.
+In this article, we discuss them and show you how they work together to
+make the communications between database and application possible. The
+concepts covered in this article will provide you with good foundation
+for applying them to your next Kubernetes full-stack application
+deployment project.  
+
+Kubernetes Persistent Volume
+----------------------------
+
+Let’s start off by explaining what a **Persistent Volume** (**PV**) is
+and what role does it play in the [Kubernetes
+Pods](https://coding-bootcamps.com/blog/kubernetes-evolution-from-virtual-servers-and-kubernetes-architecture.html).
```

Comment:
For Pod, I recommend linking to https://kubernetes.io/docs/concepts/workloads/pods/

Reply:
I agree